src/openrct2/config/Config.cpp

```diff
@@ -265,6 +264,12 @@ namespace OpenRCT2::Config
             model->FileBrowserShowSizeColumn = reader->GetBoolean("file_browser_show_size_column", true);
             model->FileBrowserShowDateColumn = reader->GetBoolean("file_browser_show_date_column", true);
             model->FileBrowserShowPreviews = reader->GetBoolean("file_browser_show_previews", true);
+
+            // Used to be an option but is now removed, SoftwareWithHardwareDisplay substitutes for Software.
+            if (model->DrawingEngine == DrawingEngine::SoftwareDeprecated)
```

Comment:
You've removed `"SOFTWARE"` from the `Enum_DrawingEngine` enum, so I think this state is no longer reachable. Probably best to restore the enum value to make the conversion work, right?

Reply:
As far as OpenRCT2 is concerned, a value of `SOFTWARE` is now just as wrong as e.g. `GUINEAPIG` and will default to whatever is the default. So as long as that default is hardware display, it will automatically change. You can just remove the `Software` case from `DrawingEngine` outright.

Reply:
> As far as OpenRCT2 is concerned, a value of `SOFTWARE` is now just as wrong as e.g. `GUINEAPIG` and will default to whatever is the default. So as long as that default is hardware display, it will automatically change. You can just remove the `Software` case from `DrawingEngine` outright.

Good point, wasn't aware of this, changed it.